template.tf

```diff
@@ -60,8 +60,15 @@ resource "oci_core_instance_configuration" "k3s_server_template" {
         image_id    = var.os_image_id
         source_type = "image"
       }
+
     }
   }
+
+  lifecycle {
```

Comment:
I added this here so it doesn't try to recreate the cluster for every change to the scripts, that's why I was thinking in managing helm from terraform but given the setup it can be a bit painful to setup, let me know if this is ok for you

Reply:
This is OK for the "always free account" but in a "on demand/paid account" but if the metadata change I want that terraform update template and my running instances.

Reply:
Okay, yeah that's fine, the only way to do this is using to resources and a variable, I have updated the code and the plan looks fine to me (either true or false)

Reply:
Removed change in the last commit